test/test_transform_ops.py

```diff
@@ -0,0 +1,265 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import contextlib
+import os
+
+os.environ["TORCH_LOGS"] = "output_code"
+import json
+import subprocess
+
+import pytest
+
+import torch
+
+from torchcodec._core import (
+    _add_video_stream,
+    add_video_stream,
+    create_from_file,
+    get_frame_at_index,
+    get_json_metadata,
+    get_next_frame,
+)
+
+from torchvision.transforms import v2
+
+from .utils import assert_frames_equal, NASA_VIDEO, needs_cuda
+
+torch._dynamo.config.capture_dynamic_output_shape_ops = True
+
+
+class TestVideoDecoderTransformOps:
```

Comment:
Nit, feel free to ignore as this is mostly personal preference, but I liked having all ops-related tests in a single file so we could just do `pytest test/test_ops.py`. Now we have to specify more files.

I definitely agree that separating the new tests in their own new test class is worth it. It these were still in in `test_ops.py`, we could run the transform-specific tests with just `pytest test/test_ops.py -k transform`